release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1014,6 +1014,36 @@ The root cause of this issue is unknown and no workaround currently exists.
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1913279[*BZ#1913279*])
 
+* Installing a cluster on {rh-openstack} that uses Kuryr times out with the following messages during bootstrapping: 
++
+[source,terminal]
+----
+INFO Waiting up to 20m0s for the Kubernetes API at https://api.ostest.shiftstack.com:6443...
+INFO API v1.20.0+ba45583 up
+INFO Waiting up to 30m0s for bootstrapping to complete...
+ERROR Attempted to gather ClusterOperator status after wait failure: listing ClusterOperator objects: Get "https://api.ostest.shiftstack.com:6443/apis/config.openshift.io/v1/clusteroperators": dial tcp 10.46.44.166:6443: connect: connection refused
+INFO Use the following commands to gather logs from the cluster
+INFO openshift-install gather bootstrap --help
+FATAL failed to wait for bootstrapping to complete: timed out waiting for the condition
+----
+
+The time out is caused by changes in how Kuryr detects the {rh-openstack} Networking service (neutron) subnet of the cluster's nodes. 
+
+As a workaround, do not remove the control plane machine manifests as described by the "Creating the Kubernetes manifest and Ignition config files" section in the installation documentation. When you are instructed to run the following command:
```

Comment:
I don't know if you want to, but since the RN is an assembly you can direct link to that reference if you like